lib/bolt/plan_creator.rb

```diff
@@ -138,5 +188,27 @@ def self.puppet_plan(plan_name)
         }
       PUPPET
     end
+
+    # Template for a new Puppet plan that only runs a script
+    # @param plan_name [string] The name of the new plan
+    # @param script [string] A reference to the script to run
+    #
+    private_class_method def self.puppet_script_plan(plan_name, script)
+      <<~PUPPET
+        # This is the structure of a simple plan. To learn more about writing
+        # Puppet plans, see the documentation: http://pup.pt/bolt-puppet-plans
+
+        # The summary sets the description of the plan that will appear
+        # in 'bolt plan show' output. Bolt uses puppet-strings to parse the
+        # summary and parameters from the plan.
+        # @summary A plan created with bolt plan new.
+        # @param targets The targets to run on.
+        plan #{plan_name} (
+          TargetSpec $targets
+        ) {
+          return run_script(#{script}, $targets)
```

Comment:
```suggestion
          return run_script('#{script}', $targets)
```

Needs to be quoted or the plan fails.